### ISSUE TITLE
[AdaptiveCards- Designer]: Focus Order is not in sequential order for the header controls on the 'Designer Page'.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7644]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:

OS: Version 21H2 (OS Build 22000.318)

URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)

Edge Dev: version 105.0.1300.0 (Official build) dev (64-bit)

Repro Steps:

1. Open above URL with valid credential.

2. Navigate to 'Designer' link and press enter key to activate it.

3. Navigate to 'Redo' button and press tab key and observe the issue.

Actual Result:

While navigating to the 'Designer' page focus order is not in sequential order on the header control as after the 'Redo' button on pressing the Tab key focus is going on the 'Copy card Payload' button.

Expected Result:

Focus order should be in logical order and after the 'Redo' button on pressing Tab key focus should be land on the Target Version combo box.

User Experience:

It will impact the keyboard user if the focus order is not in logical order as user have to navigate more to interact with desired elements.

MAS Reference:
[Focus Order - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.04.03)

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/104482612/177283476-cd7ae865-a7b5-43e4-91b7-c6d9663beb87.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by anna-dingler** [comment]
The header is broken into two `acd-toolbar-content` sections. The second section contains the target version, help button, and feedback buttons, which is why they are navigated to last.

It is more apparent here, where the second section is clearly aligned to the right of the page:

<img width="1207" alt="image" src="https://user-images.githubusercontent.com/98650930/182484764-6e3980c1-30b7-47ab-ab7b-43214ed453b2.png">

Is this behavior acceptable?

**Comment by Susmitha474** [comment]
Agree with comment, hence closing this bug.
